### ISSUE TITLE
Wrong link to license

### ISSUE THREAD

**Issue by WinterSilence** [koseven/koseven#349]
https://koseven.ga/LICENSE.md - 404, need replace to https://koseven.ga/LICENSE

**Comment by WinterSilence** [comment]
Also new classes uses:
~~~
@author    Koseven Team
@copyright (c) Koseven Team
~~~
old:
~~~
@author    Kohana Team
@copyright (c) Kohana Team
~~~

**Comment by toitzi** [comment]
@WinterSilence Agree....will take a closer look on the weekend

**Comment by toitzi** [comment]
Fixed in #372
